stimuli_ci_generation/src/server/main.py

```diff
@@ -74,12 +72,8 @@ def index():
             # and the images pass facial detection
             try:
                 generate_stimuli(participant_id, file_name)
-                pub_msg(
-                    "Completed",
-                    SIE_IMG_PROCESSING_RESULT,
-                    participant_id,
-                    experiment_id,
-                )
+                update_user_doc(participant_id, experiment_id, "completed")
```

Comment:
participant_id and experiment_id has been replaced by identifier and so should they be here. This caused some problem but I've made the change in my branch.

Reply:
our existing code has both identifier (`user_id`) and `participant_id` and `experiment_id`. Actually there was a bug which I've just fixed on line 74. We can merge your changes first if necessary then I can merge conflicts?